Source/Core/VideoCommon/RenderBase.h

```diff
@@ -133,6 +134,8 @@ class Renderer
 	// Max height/width
 	virtual int GetMaxTextureSize() = 0;
 
+	static Common::Event s_screenshotCompleted;
```

Comment:
I'd say it does, but I also think using an ```Event``` or something like that is more elegant than a ```volatile bool```.